doc/oper.xml

```diff
@@ -1964,3 +1964,162 @@ gap> ModularProduct(NullDigraph(0), CompleteDigraph(10));
+<immutable empty digraph with 0 vertices>
+]]></Example>
+</Description>
+</ManSection>
+<#/GAPDoc>
+
+<#GAPDoc Label="HomomorphicProduct">
+<ManSection>
+  <Oper Name="HomomorphicProduct" Arg="D1, D2"/>
+  <Returns>A digraph.</Returns>
+  <Description>
+  If <A>D1</A> and <A>D2</A> are symmetric digraphs without multiple edges,
+  then <C>HomomorphicProduct</C> calculates the <E>homomorphic product digraph</E>
+  (<C>HPD</C>) of <A>D1</A>and <A>D2</A>.
+
+  <C>HPD</C> has vertex set <C>V1 x V2</C> where <C>V1</C> is the vertex set of
+  <A>D1</A> and <C>V2</C> is the vertex set of <A>D2</A> (a vertex
+  <C>[a, b]</C> has label <C>(a - 1) * |V2| + b</C> in the  output). There is
+  an edge from <C>[a, b]</C> to <C>[c, d]</C> when at least one of the
+  following two conditions are satisfied:
+  <List>
+    <Item>
+      The vertices <C>a</C> and <C>c</C> of <A>D1</A> are equal.
+    </Item>
+    <Item>
+      There is an edge from <C>a</C> to <C>c</C> in <A>D1</A> and no edge from
+      <C>b</C> to <C>d</C> in <A>D2</A>.
+    </Item>
+  </List>
+
+  <Example><![CDATA[
+gap> HomomorphicProduct(PetersenGraph(),
+DigraphSymmetricClosure(ChainDigraph(4)));
+<immutable digraph with 40 vertices, 460 edges>
+gap> D1 := Digraph([[2], [1, 3, 4], [2, 5], [2, 5], [3, 4]]);                                            
+<immutable digraph with 5 vertices, 10 edges>
+gap> D2 := Digraph([[2], [1, 3], [2, 4], [3]]);                                                          
+<immutable digraph with 4 vertices, 6 edges>
+gap> HomomorphicProduct(D1, D2);                                                   
+<immutable digraph with 20 vertices, 180 edges>
+]]></Example>
+</Description>
+</ManSection>
+<#/GAPDoc>
+
+<#GAPDoc Label="LexicographicProduct">
+<ManSection>
+  <Oper Name="LexicophicProduct" Arg="D1, D2"/>
+  <Returns>A digraph.</Returns>
+  <Description>
+  If <A>D1</A> and <A>D2</A> are symmetric digraphs without multiple edges,
+  then <C>LexicographicProduct</C> calculates the <E>lexicographic product digraph</E>
+  (<C>LPD</C>) of <A>D1</A>and <A>D2</A>.
+
+  <C>LPD</C> has vertex set <C>V1 x V2</C> where <C>V1</C> is the vertex set of
+  <A>D1</A> and <C>V2</C> is the vertex set of <A>D2</A> (a vertex
+  <C>[a, b]</C> has label <C>(a - 1) * |V2| + b</C> in the  output). There is
+  an edge from <C>[a, b]</C> to <C>[c, d]</C> when at least one of the
+  following two conditions are satisfied:
+  <List>
+    <Item>
+      There is an edge from <C>a</C> to <C>c</C> in <A>D1</A>.
+    </Item>
+    <Item>
+      The vertices <C>a</C> and <C>c</C> of <A>D1</A> are equal and there is
+      an edge from <C>b</C> to <C>d</C> in <A>D2</A>.
+    </Item>
+  </List>
+
+  <Example><![CDATA[
+gap> LexicographicProduct(DigraphSymmetricClosure(CycleDigraph(3)),
+DigraphSymmetricClosure(ChainDigraph(2)));
+<immutable digraph with 6 vertices, 30 edges>
+gap> OutNeighbours(last);
+[ [ 2, 3, 4, 5, 6 ], [ 1, 3, 4, 5, 6 ], [ 1, 2, 4, 5, 6 ], [ 1, 2, 3, 5, 6 ], \
```

Comment:
I suspect the Azure CI will not like this line. You probably shouldn't have a backslash at the end, and sometimes when an output is over several lines you need to be careful with the line break characters.

If you experience CI issues with this example, the following worked for me a few weeks ago:

- Set your GAP console window to have width 72 characters (this is the width of Azure's virtual runner, I think)
- Run `LogTo("~/path/to/myfile.log");`, this will save all input and output to the specified file
- Run your example
- quit GAP
- Open the log file and copy and paste it EXACTLY into this XML file
- pray